File: geek-university-algoritmo-e-logica-de-programacao/java/exercicio_15.java
import java.util.Scanner;

public class exercicio_15 {

    public static void main(String[] args) {
        int numero;
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe um numero: ");
        numero = teclado.nextInt();

        if (numero > 100) {
            System.out.println(numero);
        } else {
            numero = 0;
            System.out.println(numero);
        }

        teclado.close();
    }

}

File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_15.c
#include <stdio.h>

int main() {
    int numero;

    printf("Informe um numero: ");
    scanf("%d", &numero);

    if (numero > 100) {
        printf("%d", numero);
    } else {
        numero = 0;
        printf("%d", numero);
    }
}
